qiskit_quantuminspire/qi_results.py

```diff
@@ -1,43 +1,72 @@
-from typing import List
-
-from compute_api_client import Result as RawJobResult
 from qiskit.providers import JobV1 as Job
+from qiskit.qobj import QobjExperimentHeader
 from qiskit.result.models import ExperimentResult, ExperimentResultData
 from qiskit.result.result import Result
 
 
 class QIResult:
     """Handle QuantumInspire (batch job) results to integrate with Qiskit's Result interface."""
 
-    def __init__(self, qinspire_results: List[RawJobResult]) -> None:
-        self._raw_results = qinspire_results
-
-    def process(self, job: Job) -> Result:
-        """Process the raw job results obtained from QuantumInspire.
+    def __init__(self, job: Job) -> None:
```

Comment:
Shouldn't this be a QIJob instead of a Qiskit Job?

Reply:
The entire class is removed